pkgs/top-level/all-packages.nix

```diff
@@ -18578,7 +18574,9 @@ with pkgs;
     ffmpeg = ffmpeg_5;
   };
 
-  ffmpegthumbnailer = callPackage ../development/libraries/ffmpegthumbnailer { };
+  ffmpegthumbnailer = callPackage ../development/libraries/ffmpegthumbnailer {
+    ffmpeg = ffmpeg-headless;
+  };
```

Comment:
Nit: I typically prefer making the package itself explicitly depend on the package variant; tweaking its inputs.

Reply:
Done. I've updated `navidrome` and `unpaper` too.